pkg/apis/admissionregistration/types.go

```diff
@@ -302,4 +300,10 @@ type ServiceReference struct {
 	// this service.
 	// +optional
 	Path *string
+
+	// If specified, the port on the service that hosting webhook.
+	// Default to 443 for backward compatibility.
```

Comment:
drop the default comment from the internal version, defaults only apply to external versions

Reply:
done